tests/deploy/values/ci-eks-config-2.yml

```diff
@@ -0,0 +1,23 @@
+elasticsearch:
+  volumeClaimTemplate:
+    storageClassName: gp2
+
+image:
+  repository: 704479110758.dkr.ecr.eu-west-2.amazonaws.com/cloudbeat-test
+  pullPolicy: Always
+
+cloudbeat:
+  deploy: false
+
+cloudbeatEks:
+  deploy: true
+
+standaloneAgent:
+  deploy: false
+
+testData:
+  marker: pre_merge
+  agentName: cloudbeat
+  clusterType: eks
+  markerCmd: -k
```

Comment:
I thought it was `-m`, pretty confusing, can we use a full notation or add a comment?

Reply:
same for -k

Reply:
`-k` command is used to select tests based on substring match, and `-m` command is used for exact match. I've added comment in this specific file.
All options are explained in the [pytest documentation](https://docs.pytest.org/en/7.1.x/example/markers.html)